pkg/sql/compile/scope.go

```diff
@@ -631,9 +645,9 @@ func (s *Scope) handleRuntimeFilter(c *Compile) error {
 					continue
 				case process.RuntimeFilter_DROP:
 					// FIXME: Should give an empty "Data" and then early return
-					s.NodeInfo.Data = nil
-					s.NodeInfo.NeedExpandRanges = false
-					s.DataSource.FilterExpr = plan2.MakeFalseExpr()
+					// s.NodeInfo.Data = nil
```

Comment:
Do we really want to remove these?   Why comment them out?   This is very confusing -- must be carefully documented, or, completely removed.

Reply:
fixed